_posts/2022-08-10-announcing-ballerina-2201.2.0-swan-lake-update-2.md

```diff
@@ -0,0 +1,122 @@
+- Now, Ballerina comes with support for running new strands safely on separate threads. The isolated feature has been extended to identify cases where strands created by a start action or a named worker can be run safely on separate threads. The strand created by a named worker can run on a separate thread from the default worker if the body of the worker satisfies the requirements for an isolated function.
+
+## Improvements to the language
+
+- The contextually-expected type for interpolations in XML template expressions has been changed to `xml`. This ensures that a query expression as an interpolation works as expected. However, it is not an error for the static type to not be a subtype of the contextually-expected type.
+- Query expressions are now allowed with `readonly` contextually-expected types. Such query expressions create immutable structural values.
+- Previously, a variable needed to be declared in the on-fail clause (e.g., `on fail error err`). It has now been made optional.
+- The grammar now allows an action such as a remote method call to occur inside a query.
+- The `lang.function:call()` langlib function has been introduced to call a function dynamically passing a function value and optional argument(s). If the arguments do not match the parameters expected by the function, the function call results in a panic.
+
+More precise explanations are provided in the [release note](https://ballerina.io/downloads/swan-lake-release-notes/swan-lake-2201.2.0).
+
+This update release comes with multiple new features and improvements in the runtime.
+
+- You can now get the status of strands and strand groups during the execution of a Ballerina program, which can be used to troubleshoot runtime errors. The Ballerina runtime will emit the strand dump to the standard output stream in the text format if it receives a `SIGTRAP` signal (`SIGTRAP` is not available on Windows).
+- You can also register a function that will be called during a graceful shutdown. A call to `onGracefulStop` will result in one call to the handler function that was passed as an argument; the handler functions will be called after calling `gracefulStop` on all registered listeners in the reverse order of the corresponding calls to `onGracefulStop` (e.g., the function foo can be called during the graceful shutdown by registering it as follows: `runtime:onGracefulStop(foo);`).
+- In terms of improvements, when a type is defined referring to another type, it will now be passed to the runtime as a `BTypeReferenceType` instance. A few existing runtime APIs have been modified to return the `BTypeReferenceType` instances.
+
+The following standard library and extended library modules now come with new features and improvements: 
```

Comment:
Shouldn't we have a subtitle like `Improvements to the standard library` to be consistent with the previous subtitle of `Improvements to the language`?

Reply:
Added.